ticketconfig.py

```diff
@@ -38,7 +38,7 @@ def _setup_providers(self):
         p.append( h.TicketHtmlTitleProvider( 'trac.torproject.org',
             'https://trac.torproject.org/projects/tor/ticket/',
             fixup=h.ReGroupFixup('.*?\((.*)\).*? Tor Bug Tracker & Wiki$'),
-            prefix='tor',
+            prefix='trac:',
             postfix=' - https://bugs.torproject.org/%s',
             default_re=r'(?<!\w)(?:[tT]or#|https://trac.torproject.org/projects/tor/ticket/)([0-9]{4,})(?:(?=\W)|$)',
             status_finder = h.TracStatusExtractor
```

Comment:
the prefix is what we write out, and it probably should match what we trigger on, so that it doesn't react to trac:#123 and then writes tor#123 issue about fu

Reply:
Als, just "trac" is a bit unspecific.  Remember that the bot is not only used by torproject

Reply:
tortrac then maybe?